### ISSUE TITLE
Display author field (no dropdown) when site has only one user

### ISSUE THREAD

**Issue by garretthyder** [WordPress/gutenberg#17725]
**Describe the bug**
With the Classic Editor the Author metabox was available with the dropdown even if only a single author on site. With Gutenberg the author information isn't present in the editor unless you have more than one author available. It would be nice to provide the author information even if just one users maybe as just their name rather than a select as they can't be changed. This would allow authors on new posts to see what their username is before making the post.

**To reproduce**
1. In a clean install with one user go to create/edit a post and try to find author information

**Expected behavior**
The author information should always be present even if there's only a single author.

**Screenshots**
<img width="1122" alt="Screen Shot 2019-10-02 at 10 55 28 AM" src="https://user-images.githubusercontent.com/8726005/66069369-149d6500-e504-11e9-86be-08002e1a90a2.png">

**VERSIONS:**
 - Desktop OS: macOS Mojave 10.14.5
 - Browser: Chrome 76.0.3809.132
 - WordPress: 5.2.3
 - Gutenberg: Core5.2.3

**Comment by paaljoachim** [comment]
I believe this has been fixed and will close this issue. If I am mistaken then please reopen.

**Comment by garretthyder** [comment]
👋  @paaljoachim 
Checking on the latest WP 5.6 with and without the latest Gutenberg plugin installed I still find the Author information is only ever exposed once you have more than one author.

Single Author;
![Screen Shot 2021-01-13 at 2 57 01 PM](https://user-images.githubusercontent.com/8726005/104520638-4c70d480-55b0-11eb-94c6-5aab8ec20417.png)

Multi-Author;
![Screen Shot 2021-01-13 at 2 56 25 PM](https://user-images.githubusercontent.com/8726005/104520646-50045b80-55b0-11eb-8e4d-714755f2edc0.png)

In single view we should still display the Author name even though there's no other authors to switch to.

Can you re-open Paal? I don't have permissions to reopen the issue.

**Comment by paaljoachim** [comment]
Hey Garrett. Thanks for checking!

@adamsilverstein I think this issue is just up your alley so to speak.
We should have Author name listed even when there is just the one author.

**Comment by adamsilverstein** [comment]
That is by design, although I'm not sure why the decision was made that it should work that way. I guess because in a one user system you are the only user and you cannot change the user.

I wonder how this behaved in the classic editor, does this match the behavior we have always had?

I can see how this might not need to show for the vast majority of sites where there is only one user: they never see a dropdown because they don't even think in terms of "Authors" - they are the only author on the site.

**Comment by paaljoachim** [comment]
In regards to the Classic Editor with one author:

![Screen Shot 2021-01-14 at 00 17 30](https://user-images.githubusercontent.com/5323259/104521833-f62a8280-55fd-11eb-90a1-3926630af7dc.png)

**Comment by adamsilverstein** [comment]
Ah, I see in the description the dropdown showed previously showed even in one user systems. I would feel fine showing just the Author name here, curious what the design feedback is.

**Comment by kellychoffman** [comment]
A dropdown with a single select would make for a frustrating UX. I say we keep it as is.

This is not to say we can't consider other improvements, but I think we can open up separate issues for those.

**Comment by paaljoachim** [comment]
I am thinking instead of a drop down just listing the Author and the name like so:
![Screen Shot 2021-01-14 at 00 54 40](https://user-images.githubusercontent.com/5323259/104526608-20327380-5603-11eb-9d43-5769e362335a.png)

I agree there is no need for a drop down with only one name, but it would be nice to just list the author name.

**Comment by kellychoffman** [comment]
I don't think it is valuable unless there's a link or indication of how to change it , which requires a bit more thought and a small flow designed before we make the change IMO.

**Comment by garretthyder** [comment]
Mostly here am thinking of new users so they see their username during post creation and realize they should change it from `admin`.

**Comment by manzwebdesigns** [comment]
I think users should be warned if they are using a common username during the user setup.  Personally, I also think there is no need to make the username visible if there is only one... how often do people not know what their's is?  My guess is very few.

**Comment by noisysocks** [comment]
What do you think about always showing the Author dropdown @javierarce? Personally I think it's unnecessary.

**Comment by paaljoachim** [comment]
I got a feeling that we should close this issue as a wont fix. 
I do fint it helpful to see my own username in the Publish screen. It is a reminder about the username I have on a site. But I can also let it go.

**Comment by javierarce** [comment]
I agree with the opinion that a dropdown with just one value is a frustrating UI, so I'm in favor of just showing the username in that case.

**Comment by Mamaduka** [comment]
The document settings went through a couple of refactorings, which I think resolved this issue. The dropdown has been moved into a popover, and it's less frustrating, even with one user.

## Screenshot

![Image](https://github.com/user-attachments/assets/7e26f8a5-cbc0-4b1c-9dbc-eea0683fe85e)
